pkg/constants/constants.go

```diff
@@ -27,9 +27,63 @@ const (
 	// UninstallJobLabel is the label used for artifacts specific to Hive cluster deprovision.
 	UninstallJobLabel = "hive.openshift.io/uninstall"
 
-	// ClusterDeploymentNameLabel is the label that is used to identify the installer pod of a particular cluster deployment
+	// ClusterDeploymentNameLabel is the label that is used to identify a relationship to a given cluster deployment object.
 	ClusterDeploymentNameLabel = "hive.openshift.io/cluster-deployment-name"
 
+	// ClusterDeprovisionNameLabel is the label that is used to identify a relationship to a given cluster deprovision object.
+	ClusterDeprovisionNameLabel = "hive.openshift.io/cluster-deprovision-name"
+
+	// ClusterProvisionNameLabel is the label that is used to identify a relationship to a given cluster provision object.
+	ClusterProvisionNameLabel = "hive.openshift.io/cluster-provision-name"
+
+	// PVCTypeLabel is the label that is used to identify what a PVC is being used for.
+	PVCTypeLabel = "hive.openshift.io/pvc-type"
+
+	// PVCTypeInstallLogs is used as a value of PVCTypeLabel that says the PVC is specifically stores installer logs.
+	PVCTypeInstallLogs = "installlogs"
+
+	// JobTypeLabel is the label that is used to identify what a Job is being used for.
+	JobTypeLabel = "hive.openshift.io/job-type"
+
+	// JobTypeImageSet is used as a value of JobTypeLabel that says the Job is specifically running to determine which imageset to use.
+	JobTypeImageSet = "imageset"
+
+	// JobTypeDeprovision is used as a value of JobTypeLabel that says the Job is specifically running the deprovisioner.
+	JobTypeDeprovision = "deprovision"
+
+	// JobTypeProvision is used as a value of JobTypeLabel that says the Job is specifically running the provisioner.
+	JobTypeProvision = "provision"
+
+	// DNSZoneTypeLabel is the label that is used to identify what a DNSZone is being used for.
+	DNSZoneTypeLabel = "hive.openshift.io/dnszone-type"
+
+	// DNSZoneTypeTargetCluster is used as a value of DNSZoneTypeLabel that says the DNSZone is specifically used as the forwarding zone for the target cluster.
+	DNSZoneTypeTargetCluster = "targetcluster"
```

Comment:
What do folks think of "forwarding" for the value here instead.

Reply:
@dgoodwin we can definitely do that. Just so it's clear, this is the dns zone that is forwarded to from the main "parent" zone.

So maybe `forwarded` would be clearer?

Here are some other alternatives that I thought of:
- `managedcluster`
- `target`
- `child` # since the zone forwarding to it is called the `parent` zone

Reply:
'child' seems reasonable since we already have the 'LinkToParent' terminoligy

Reply:
@dgoodwin I'm ready to change this to whatever your decision is.

Reply:
"child" works for me. Thanks!